src/bigint.hpp

```diff
@@ -31,115 +31,111 @@
 #include <iostream>
 #include <string>
 
-namespace libbig {
```

Comment:
please keep following this
```C++
void function () {
}
namespace {
}
int main () {
}
```
pattern instead of
```C++
int main ()
{
}
void function() 
{
}
```
this pattern.
soon we will add contribution and coding guidelines to this repository